exercises/pig-latin/test/Tests.hs

```diff
@@ -10,14 +10,15 @@ specs :: Spec
 specs = describe "pig-latin" $
           describe "translate" $ do
 
-    -- Test cases adapted from `exercism/x-common/pig-latin.json` on 2016-09-08.
+    -- Test cases adapted from `exercism/x-common/pig-latin.json` on 2016-11-06.
 
     describe "ay is added to words that start with vowels" $ do
         it "word beginning with a" $ translate "apple"  `shouldBe` "appleay"
         it "word beginning with e" $ translate "ear"    `shouldBe` "earay"
         it "word beginning with i" $ translate "igloo"  `shouldBe` "iglooay"
         it "word beginning with o" $ translate "object" `shouldBe` "objectay"
         it "word beginning with u" $ translate "under"  `shouldBe` "underay"
+        it "word beginning with a vowel and followed by a qu" $ translate "equal" `shouldBe` "equalay"
```

Comment:
nah. I'll move it to after the "square" test, that's where it makes th emost sense.